src/public/css/index.css

```diff
@@ -0,0 +1,107 @@
+#title {
+	text-align: center;
+	font-weight: bold;
+	font-size: 40px;
+	font-family: "Montserrat";
+}
+
+#timeline-div {
+	margin-right: 40px;
+	margin-left: 40px;
+	margin-bottom: 20px;
+}
+
+#update-timeline {
+	text-align: left;
+	margin-bottom: 4px;
+}
+
+#tweets {
+	margin-top: 10px;
+	border-style: solid;
+	border-width: 2px;
+	font-family: "Times New Roman";
+}
+
+.tweet-link {
+	text-decoration: none;
+}
+
+.tweet {
+	display: inline-grid;
+	grid-template-areas: "user-grid content-grid";
+	color: black;
+	padding-bottom: 20px;
+	padding-top: 20px;
+	padding-right: 10px;
+	align-items: center;
+}
+
+.user-div {
+	text-align: center;
+	grid-area: user-grid;
+	width: 160px;
+}
+
+.profile-image {
+	border-radius: 50%;
+	width: 80px;
+}
+
+.display-name {
+	margin-top: 2px;
+	overflow: hidden;
+	justify-content: center;
+	font-size: 15px;
+	font-weight: bold;
+	padding-top: 2px;
+	padding-bottom: 2px;
+	padding-left: 10px;
+	max-width: 140px;
+}
+
+.twitter-handle {
+	margin-right: 10px;
+	overflow: hidden;
+	justify-content: center;
+	margin-top: 1px;
+	font-size: 12px;
+	padding-left: 10px;
+	max-width: 140px;
+}
+
+.content-div {
+	align-self: center;
+	grid-area: content-grid;
+	min-width: 0px;
+	overflow: hidden;
+}
+
+.date {
+	font-style: italic;
+	font-size: 11px;
+}
+
+.message {
+	margin-top: 6px;
+	font-size: 12px;
+}
+
+.odd-row {
+	border-bottom: solid;
```

Comment:
single line for border, looks like this is shared between odd and even row styles...combine the 2 where possible